blocks_vertical/sound.js

```diff
@@ -40,27 +34,11 @@ Blockly.Blocks['sound_sounds_menu'] = {
           "type": "field_dropdown",
           "name": "SOUND_MENU",
           "options": [
-            ['1', '0'],
-            ['2', '1'],
-            ['3', '2'],
-            ['4', '3'],
-            ['5', '4'],
-            ['6', '5'],
-            ['7', '6'],
-            ['8', '7'],
-            ['9', '8'],
-            ['10', '9'],
-            ['call a function', function() {
-              window.alert('function called!');}
-            ]
```

Comment:
AFAICT this is vestigial debugging code; the menu gets patched by scratch-gui here: https://github.com/scratchfoundation/scratch-gui/blob/develop/src/lib/blocks.js#L158-L161

Reply:
It looks like we actually don't need any of the code for this block then? We just need to make sure `sound_sounds_menu` is defined.

(Also I feel like there's probably a better way to do this with a dynamic drop-down, but I don't think we need to do that.)

Reply:
Good point, removed accordingly.